clidocstool_yaml.go

```diff
@@ -169,6 +169,9 @@ func (c *Client) genYamlCustom(cmd *cobra.Command, w io.Writer) error {
 
 	// check recursively to handle inherited annotations
 	for curr := cmd; curr != nil; curr = curr.Parent() {
+		if curr.Hidden {
+			cliDoc.Hidden = true
+		}
```

Comment:
Wondering if we should keep these for the YAML docs (being more "data" then "end-result")?

Reply:
The YAML file is still generated. What's changed is hidden attribute is now inherited from parent if set: https://github.com/docker/cli-docs-tool/pull/65/files#diff-cce07c459f10c3f29f937920cb8ba11cc3ab7028890917e036b3241da8940febR28

Reply:
ah! Sorry ignore me then 🤗